nextcord/http.py

```diff
@@ -924,6 +982,79 @@ def start_thread_without_message(
 
         route = Route('POST', '/channels/{channel_id}/threads', channel_id=channel_id)
         return self.request(route, json=payload, reason=reason)
+    
+    def start_thread_in_forum_channel(
+        self,
+        channel_id: Snowflake,
+        *,
+        name: str,
+        auto_archive_duration: threads.ThreadArchiveDuration,
+        rate_limit_per_user: int,
+        content: Optional[str] = None,
+        embed: Optional[embed.Embed] = None,
+        embeds: Optional[List[embed.Embed]] = None,
+        nonce: Optional[str] = None,
+        allowed_mentions: Optional[message.AllowedMentions] = None,
+        stickers: Optional[List[sticker.StickerItem]] = None,
+        components: Optional[List[components.Component]] = None,
+        reason: Optional[str] = None,
+    ):
```

Comment:
Typehing return type here as well

Reply:
resolved in `eec0812`, typehinted with Response[threads.Thread] - as the coroutines above.